context/AuthContextProvider.tsx

```diff
@@ -100,7 +100,7 @@ export const AuthContextProvider = ({
       resetUser();
```

Comment:
As always, your beautiful pieces of code gone. :) I'd love to know how you found out `useCallback` was causing more rendering. :)